testdata/p4_16_samples/ipv4_forward-bmv2.stf

```diff
@@ -0,0 +1,8 @@
+set_default ipv4_lpm drop
```

Comment:
before checking these in, we should make sure they are correct. I'm not convinced that the packet data is going to work for this test.